proto/sequencerblockapis/astria/sequencerblock/v1alpha1/block.proto

```diff
@@ -0,0 +1,136 @@
+syntax = "proto3";
+
+package astria.sequencerblock.v1alpha1;
+
+import "astria/primitive/v1/types.proto";
+import "astria/sequencerblock/v1alpha1/merkle.proto";
+import "google/protobuf/timestamp.proto";
+
+// `RollupTransactions` are a sequence of opaque bytes together with a 32 byte
+// identifier of that rollup.
+//
+// The binary encoding is understood as an implementation detail of the
+// services sending and receiving the transactions.
+message RollupTransactions {
+  // The 32 bytes identifying a rollup. Usually the sha256 hash of a plain rollup name.
+  bytes id = 1;
+  // The serialized bytes of the rollup data.
+  // Each entry is a protobuf-encoded `RollupData` message.
+  repeated bytes transactions = 2;
+  // The proof that these rollup transactions are included in sequencer block.
+  // `astria.sequencer.v1alpha.SequencerBlock.rollup_transactions_proof`.
+  astria.sequencerblock.v1alpha1.Proof proof = 3;
+}
+
+// `SequencerBlock` is constructed from a tendermint/cometbft block by
+// converting its opaque `data` bytes into sequencer specific types.
+message SequencerBlock {
+  // the block header, which contains sequencer-specific commitments.
+  SequencerBlockHeader header = 1;
+  // The collection of rollup transactions that were included in this block.
+  repeated RollupTransactions rollup_transactions = 2;
+  // The proof that the rollup transactions are included in the CometBFT block this
+  // sequencer block is derived form. This proof together with
+  // `Sha256(MTH(rollup_transactions))` must match `header.data_hash`.
+  // `MTH(rollup_transactions)` is the Merkle Tree Hash derived from the
+  // rollup transactions.
+  astria.sequencerblock.v1alpha1.Proof rollup_transactions_proof = 3;
+  // The proof that the rollup IDs listed in `rollup_transactions` are included
+  // in the CometBFT block this sequencer block is derived form.
+  //
+  // This proof is used to verify that the relayer that posts to celestia
+  // includes all rollup IDs and does not censor any.
+  //
+  // This proof together with `Sha256(MTH(rollup_ids))` must match `header.data_hash`.
+  // `MTH(rollup_ids)` is the Merkle Tree Hash derived from the rollup IDs listed in
+  // the rollup transactions.
+  astria.sequencerblock.v1alpha1.Proof rollup_ids_proof = 4;
+  /// The block hash of the cometbft block that corresponds to this sequencer block.
+  bytes block_hash = 5;
+}
+
+message SequencerBlockHeader {
+  // the cometbft chain ID of the sequencer chain
+  string chain_id = 1;
+  // the height of this sequencer block
+  uint64 height = 2;
+  // the timestamp of this sequencer block
+  google.protobuf.Timestamp time = 3;
+  // the data_hash of the sequencer block (merkle root of all transaction hashes)
+  bytes data_hash = 4;
+  // the cometbft proposer address of the sequencer block
```

Comment:
Is this already affected our discussion re `bech32`? If yes, then we should probably change this to `string` because of the protobuf <> json mapping

Reply:
answered in slack - we can change this later as it's v1alpha1, and also other protos in sequencer would need to change as well, so would rather do it all at once in another PR